src/common/scheduler/policy/alternate_policy.go

```diff
@@ -0,0 +1,185 @@
+package policy
+
+import "github.com/vmware/harbor/src/common/scheduler/task"
+import "errors"
+import "time"
+
+//AlternatePolicyConfiguration store the related configurations for alternate policy.
+type AlternatePolicyConfiguration struct {
```

Comment:
Do we need this?  Given policy can be very lightweight and highly flexible, how about implement a simple "Daily" policy?